temba/channels/views.py

```diff
@@ -1844,6 +1846,59 @@ def form_valid(self, form):
 
             return HttpResponseRedirect(reverse('channels.channel_configuration', args=[channel.id]))
 
+    class ClaimLine(OrgPermsMixin, SmartFormView):
+        class LineForm(forms.Form):
+            channel_id = forms.CharField(label=_("Channel ID"), required=True, help_text=_("The ID of the your Channel on Business LINE"))
+            channel_secret = forms.CharField(label=_("Channel Secret"), required=True, help_text=_("The Secret of the your Channel on Business LINE"))
+            channel_mid = forms.CharField(label=_("Channel MID"), required=True, help_text=_("The MID of the your Channel on Business LINE"))
```

Comment:
Do they call them Channel's at LINE? If that's the case, then make these read something like:

"The MID of your Business LINE Channel"

Either way, "of the your" on all of these is not right.